man/man8/zpool.8

```diff
@@ -597,6 +597,15 @@ This space occurs when a LUN is dynamically expanded.
 The amount of fragmentation in the pool.
 .It Sy free
 The amount of free space available in the pool.
+
+Not all
+.Sy free
+space in a pool will be
+.Sy available
+to zfs volumes in the pool. Some space is reserved, for the zpool itself. See
+.Sy spa_slop_shift
+in the
+.Xr zfs-module-parameters 5
```

Comment:
This really is a substantial topic to properly cover here, but I entirely agree that it's worth adding a few lines to make it clear this is to be expected.  I'm sure this is one of the most common questions new users have.  What do you think about something like this (formatting needed, and better phrasing welcome).  These are the two biggest contributing factors to any discrepancy.

"The amount of free space available in the pool.  Not all __free__ space in the pool will be __available__ for use by filesystems and volumes.  The pool must reserve some capacity for itself, see __spa_slop_shift__ in __zfs-module-parameters(5)__ for details.  Furthermore, RAIDZ configurations must reserve capacity for the required parity data."

Reply:
That text seems ok to me. Should I incorporate it into this PR?